### ISSUE TITLE
dataset？

### ISSUE THREAD

**Issue by 13512525** [SuperMHP/GUPNet_Plus#4]
Hello, thank you for your excellent work. I noticed you commented out the training and validation set code for the nuscenes dataset in your dataset processing script. Can this code be used to train with the nuscenes training set?
